docs/modules/kafka.md

```diff
@@ -82,3 +91,34 @@ The `Brokers(ctx)` method returns the Kafka brokers as a string slice, containin
 <!--codeinclude-->
 [Get Kafka brokers](../../modules/kafka/kafka_test.go) inside_block:getBrokers
 <!--/codeinclude-->
+
+#### BrokersByHostDockerInternal
+
+The `BrokersByHostDockerInternal(ctx)` method returns the Kafka brokers as a
+string slice, containing the hostname `host.docker.internal` and a random port
+defined by Kafka's public port (`19092/tcp`).
+
+This method is useful when you need to run additional containers that need to
+connect to Kafka.
+
+<!--codeinclude-->
+[Get Kafka brokers by host.docker.internal](../../modules/kafka/examples_test.go) inside_block:getBrokersByHostDockerInternal
+<!--/codeinclude-->
+
+#### BrokersByContainerName
+
+The `BrokersByContainerName(ctx)` method returns the Kafka brokers as a string
+slice, addressed by the container's name(`Ex: charming_dijkstra:19093`). This
+method is useful when you need to run additional containers that need to connect
+to Kafka.
+
+To use this broker address you should run all the containers inside a docker
+network.
+
+<!--codeinclude-->
+[Start Kafka inside a docker network](../../modules/kafka/examples_test.go) inside_block:getBrokersByContainerName_Kafka
```

Comment:
bug: looks like wrong name here says by docker network and then links to by container.

Reply:
I think this is correct, It's a 2 part explanation, I was trying to hightlight that this works best (maybe ONLY works) if you start the Kafka testcontainer inside a non-default docker network. So I broke this into 2 codeincludes, one focusing on starting the Kafka test container in a non-default network and the second part of starting another container in that same network.